src/pages/Vaults/VaultsOverview/components/CreateVaultWizard/DisclaimerStep.tsx

```diff
@@ -0,0 +1,85 @@
+import { Trans, useTranslation } from 'react-i18next';
+
+import { ReactComponent as WarningIcon } from '@/assets/img/icons/exclamation-triangle.svg';
+import { CTA, CTALink, H3, Stack, TextLink } from '@/component-library';
+import {
+  INTERLAY_DOS_AND_DONTS_DOCS_LINK,
+  INTERLAY_TERMS_AND_CONDITIONS_LINK,
+  INTERLAY_VAULT_DOCS_LINK
+} from '@/config/links';
+
+import {
+  StyledDisclaimerCard,
+  StyledDisclaimerList,
+  StyledDisclaimerListItem,
+  StyledDisclaimerText
+} from './CreateVaultWizard.styles';
+import { StepComponentProps, withStep } from './Step';
+
+type Props = {
+  onClickAgree?: () => void;
+};
+
+type DisclaimerStepProps = Props & StepComponentProps;
+
+const DisclaimerStep = ({ onClickAgree }: DisclaimerStepProps): JSX.Element => {
+  const { t } = useTranslation();
+
+  return (
+    <Stack spacing='double'>
+      <Stack spacing='none' alignItems='center'>
+        <WarningIcon width='2.5rem' height='2.5rem' />
+        <H3>{t('vault.disclaimer.plase_read_before_you_start')}</H3>
+      </Stack>
+      <Stack>
+        <StyledDisclaimerText>{t('vault.disclaimer.understand_risks')}</StyledDisclaimerText>
+        <StyledDisclaimerText>
+          {t('vault.disclaimer.running_vault_requirments')} {t('vault.disclaimer.understand_liquidation_risks')}
+        </StyledDisclaimerText>
+        <StyledDisclaimerCard color='secondary'>
+          {t('vault.disclaimer.before_start_vault')}
+          <StyledDisclaimerList>
+            <StyledDisclaimerListItem>
+              <Trans i18nKey='vault.disclaimer.check_do_s_and_dont_s'>
+                Check the
```

Comment:
Anyway to move this text into the locale files?

Reply:
Ok - fine for now, but I think we need a way around this. We'll come back to it :)